os/fs/fs_initialize.c

```diff
@@ -82,6 +84,33 @@
  * Private Functions
  ****************************************************************************/
 
+static void fs_common_automount(void)
```

Comment:
Actually we initiate fs_initialize only for inode for now, In my opinion mount should be called after all peripheral initialized.
procfs including smartfs and something else, I think all other file's changes are acceptable, but not this one.